index.js

```diff
@@ -1,9 +1,11 @@
-import nacl from 'tweetnacl'
-import {blake2bInit, blake2bUpdate, blake2bFinal} from 'blakejs'
+'use strict';
+const tweetSodium = module.exports;
```

Comment:
Yeah -- this would be for TypeScript specifically (defining the typings), but potentially useful for JavaScript users. I'm not a pro by any means at JavaScript though.